cookbook/Handling-Persistence-Exception.md

```diff
@@ -0,0 +1,82 @@
+# Handling exception in persistence
+
+Say, you have a code like this which throws exception:
+
+``` haskell
+share [mkPersist sqlSettings, mkMigrate "migrateAll"] [persistLowerCase|
+User
+    age Int
+    UniqueAge age
+    deriving Show
+|]
+
+getUser :: MonadIO m => ReaderT SqlBackend m [Entity User]
```

Comment:
Minor recommendation: `getUsers` instead of `getUser` may be a bit easier to parse for some.